babel/src/test/resources/sql/redshift.iq

```diff
@@ -1238,7 +1238,7 @@ select date_part(minute, time '02:08:01');
 select date_part(minute, date '2009-01-01');
 !ok
 !}
-
+!if (false) {
```

Comment:
DATE_TRUNC() in redshift has the operands reversed (DATE_TRUNC(date_part, date_expression) and the babel test has it marked as LENIENT for functions that aren't implemented, so I'm commenting this out.

Reply:
This test should be re-enabled before we accept the PR. If the test is incorrect, fix it. If you need a different DATE_TRUNC function for BQ than for Redshift, that can be accomplished.

Reply:
Re-enabled!

Reply:
I know Redshift was not one of your specific goals, so I want to thank you for going the extra mile to make this work for Redshift. In Calcite, quality tends to multiply like compound interest, so thanks for playing your part.